### ISSUE TITLE
Community Shout out file 

### ISSUE THREAD

**Issue by helenwoodcock** [Tech4GoodBristol/t4gb-community-shout-outs#1]
Here are the shout-out details.

**FILE REMOVED BY ADMIN**

**Comment by jennet** [comment]
that's the guest list?

**Comment by helenwoodcock** [comment]
Yes sorry wrong file! But will upload the right file now. I'm not used to
Github!

**Comment by helenwoodcock** [comment]
[Tech4Good Launch Guestlist - Community Shout out.pdf](https://github.com/Tech4GoodBristol/community-shout-outs/files/875865/Tech4Good.Launch.Guestlist.-.Community.Shout.out.pdf)
Here's the right community shout out file

**Comment by jennet** [comment]
Do you have it as a non PDF version? CSV?

**Comment by helenwoodcock** [comment]
Yes have shared all versions via google docs so would be good if you could
collect there. I'm stacked today sorry.

**Comment by jennet** [comment]
cool no worries :-)
